internal/langserver/handlers/did_change_watched_files.go

```diff
@@ -0,0 +1,40 @@
+package handlers
+
+import (
+	"context"
+
+	lsctx "github.com/hashicorp/terraform-ls/internal/context"
+	ilsp "github.com/hashicorp/terraform-ls/internal/lsp"
+	"github.com/hashicorp/terraform-ls/internal/protocol"
+	lsp "github.com/hashicorp/terraform-ls/internal/protocol"
+)
+
+func DidChangeWatchedFiles(ctx context.Context, params lsp.DidChangeWatchedFilesParams) error {
```

Comment:
I've missed doing module-level configuration here for the passed document but since this notification spans across multiple documents, will it be a good idea to handle them based on the `FileChangeType`?